Cython/Utility/ModuleSetupCode.c

```diff
@@ -1,6 +1,10 @@
 /////////////// InitLimitedAPI ///////////////
 
-#if defined(CYTHON_LIMITED_API) && 0  /* disabled: enabling Py_LIMITED_API needs more work */
+#if defined(CYTHON_LIMITED_API) && defined(CYTHON_TEST_IN_LIMITED_API)
+  /* disabled by default: enabling Py_LIMITED_API needs more work.
+     Users should not define CYTHON_TEST_IN_LIMITED_API since it will
+     be removed when support is better.
+     If you want Py_LIMITED_API then set it yourself for now */
```

Comment:
Hmm, then why don't *we*  set `Py_LIMITED_API` in the tests and instead need a new macro guard? The `CYTHON_*` macro names are specifically meant for users to define. I would leave it to `runtests.py` to set `Py_LIMITED_API` to something reasonable.